sdk/identity/Azure.Identity/src/IdentityClientOptions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.Pipeline;
+using Azure.Core.Pipeline.Policies;
+
+namespace Azure.Identity
+{    public class IdentityClientOptions : HttpClientOptions
```

Comment:
What are we going to do with the service API versions in this options type? Our guidelines state that you should take version to the ctor.

Reply:
I need to talk to the identity team about this.  I believe currently they only have two service versions v1, and v2 and they are different endpoints.  Also I was intending on doing MSI calls through the same pipeline which is yet another endpoint, and has separate versioning.